### DISCUSSION TITLE
sync_list: exclude folder inside included folder?

### DISCUSSION THREAD

**Discussion by Fire69** [abraunegg/onedrive#2240] [Q&A] [answered]
Hi,

I'm trying to exclude 1 folder that's inside a folder defined in sync_list.

Would this be correct?

!/Pictures/USA 2022/OneRS
Pictures

When I do a --resync it still includes the folder.

**Comment by abraunegg** [comment]
@Fire69 
What application version are you using? What is the output of `onedrive --version` ?

**Edit:**
This sounds like this bug: https://github.com/abraunegg/onedrive/issues/1819 - fixed in https://github.com/abraunegg/onedrive/releases/tag/v2.4.16

**Comment by Fire69** [comment]
I'm on v2.4.11, I'll upgrade and test.  Thanks!

**Comment by abraunegg** [comment, marked as answer]
@Fire69 
> I'm on v2.4.11

That version was released Apr 7, 2021. Only the current release version or greater is supported. Earlier versions are not supported and should not be installed or used. The current client release is: [![Version](https://img.shields.io/github/v/release/abraunegg/onedrive)](https://github.com/abraunegg/onedrive/releases)

Unsure how you installed the client as well, however please ensure that keep your client current to avoid running into bugs and issues that have been fixed.

**Reply by Fire69** [reply]
Tbh, I thought I installed it using yann1ck's ppa, but when I follow your instructions to clean up the install, apt-get says the package isn't installed, so now I'm not sure what to do anymore to remove it (sorry, I'm not great with Linux).  
It appears to be installed in /usr/local/bin. Any tips on how to correctly remove it?

**Reply by abraunegg** [reply]
@Fire69 
> It appears to be installed in /usr/local/bin. Any tips on how to correctly remove it?

Unfortunately you need to seek support from whomever supports your chosen Linux distribution.
